Backend/src/ml_lib/shared/shared_model.py

```diff
@@ -64,37 +62,3 @@ def df(self) -> pd.DataFrame:
             raise Exception("Dataframe is None.")
 
         return self._df
-
-
-    def dim_red(self, embeddings: torch.Tensor, dim: str | None) -> pd.DataFrame:

```

Comment:
It would be nice to create a parent function as a template. This will make it easier to know what params to use and what to name the function. 

For this function, it could just raise an Exception("Not implemented")

Reply:
that is a good suggestion, will implement that

Reply:
done